app/src/main/java/com/kosbrother/mongmongwoo/utils/DateFormatUtil.java

```diff
@@ -7,7 +7,7 @@
 
 public class DateFormatUtil {
 
-    public static final String API_CREATED_AT_PATTERN = "yyyy-MM-dd'T'hh:mm:ss.sss";
+    public static final String API_CREATED_AT_PATTERN = "yyyy-MM-dd'T'kk:mm:ss.sss";
```

Comment:
Kk會轉成24小時制的hour 我剛剛測試發現下午12點的如果用hh他會回傳0點 查了一下說要這樣改

2016/7/30 下午2:12於 "Steven Ko" <notifications@github.com>寫道：

> In app/src/main/java/com/kosbrother/mongmongwoo/utils/DateFormatUtil.java
> <https://github.com/kosbrother/AndroidMongMong/pull/104#discussion_r72882809>
> :
>
> > @@ -7,7 +7,7 @@
> >
> >  public class DateFormatUtil {
> >
> > -    public static final String API_CREATED_AT_PATTERN = "yyyy-MM-dd'T'hh:mm:ss.sss";
> > +    public static final String API_CREATED_AT_PATTERN = "yyyy-MM-dd'T'kk:mm:ss.sss";
>
> kk 是什麼啊？
> https://developer.android.com/reference/java/text/SimpleDateFormat.html
> "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kosbrother/AndroidMongMong/pull/104/files/5907f3b8f372542f7ebaa95d047d0cfc8820ddf9..adec448a41535745ad6465bd093102beb8fb73b7#r72882809>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP9wZPJ6_5MrpZP1TsPxvJ9Dg7n-lISkks5qautFgaJpZM4JW3CY>
> .
>